data/pc/latest/proto.yml

```diff
@@ -1,4 +1,4 @@
-!version: 1.21.4
+!version: 1.21.5
```

Comment:
This seems to have no change. Is this right?

Reply:
If the proto.yml files is a yaml version of the protocol.json then alot more I would assume has to change.